src/gui/newwizard/pages/serverurlsetupwizardpage.cpp

```diff
@@ -21,6 +22,9 @@ ServerUrlSetupWizardPage::ServerUrlSetupWizardPage(const QUrl &serverUrl)
     connect(this, &AbstractSetupWizardPage::pageDisplayed, this, [this]() {
         _ui->urlLineEdit->setFocus();
     });
+
+    _ui->logoLabel->setText(QString());
+    _ui->logoLabel->setPixmap(Theme::instance()->wizardHeaderLogo().pixmap(200, 200));
```

Comment:
I wonder if 200x200 size is always sufficient... Maybe consider two other theme values for width and height? Just in case...

Reply:
We may have to test that on some high DPI system. I also think Qt is going to handle the scaling, though.